src/substrate-lib/index.ts

```diff
@@ -0,0 +1,4 @@
+import { SubstrateProvider, useSubstrate, useSubstrateState } from './substrate-context';
+import utils from './utils';
+
+export { SubstrateProvider, useSubstrate, useSubstrateState, utils };
```

Comment:
we should specify the kind of utils here:

`utils as substrateUtils`

Reply:
Done!